apps/backend/src/deposits/cash-deposit.service.ts

```diff
@@ -113,27 +110,50 @@ export class CashDepositService {
    *  pending or in progress. This is used to find the dates that need to be reconciled.
    */
   public async depositDates(event: ReconciliationEvent): Promise<string[]> {
-    const {
-      date,
-      program,
-      location: { pt_location_id }
-    } = event;
+    const { fiscal_close_date, fiscal_start_date, program, location } = event;
 
     const dates = await this.cashDepositRepo.find({
       select: { deposit_date: true },
       where: {
-        pt_location_id,
+        pt_location_id: location.pt_location_id,
         metadata: { program },
-        deposit_date: LessThanOrEqual(date)
+        deposit_date: Raw(
+          (alias) =>
+            `${alias} <= :fiscal_close_date and ${alias} > :fiscal_start_date `,
+          {
+            fiscal_close_date,
+            fiscal_start_date
+          }
```

Comment:
Get the list of dates between the fiscal_start and fiscal_close. I can't see a need to match outside of the specified date range.